composer.json

```diff
@@ -18,38 +18,38 @@
     ],
     "require": {
         "php": "^5.3 || ^7.0",
-        "symfony/console": "^2.3",
-        "symfony/form": "^2.3",
-        "symfony/framework-bundle": "^2.3",
-        "symfony/templating": "^2.3",
         "sonata-project/admin-bundle": "^3.4",
-        "sonata-project/user-bundle": "^3.0",
-        "sonata-project/media-bundle": "^3.0",
         "sonata-project/classification-bundle": "^3.0",
-        "sonata-project/formatter-bundle": "^3.0",
+        "sonata-project/core-bundle": "^3.0",
+        "sonata-project/datagrid-bundle": "^2.2.1",
         "sonata-project/easy-extends-bundle": "^2.2",
+        "sonata-project/formatter-bundle": "^3.0",
         "sonata-project/intl-bundle": "^2.2.4",
-        "sonata-project/datagrid-bundle": "^2.2.1",
-        "sonata-project/core-bundle": "^3.0",
-        "swiftmailer/swiftmailer": "^4.3 || ^5.0"
+        "sonata-project/media-bundle": "^3.6",
```

Comment:
This version constraint changes in case anyone is wondering

Reply:
Oh but this is obvious when you read commit by commit, like I should have. Great job!